docs/specification/1.0.md

```diff
@@ -496,3 +496,45 @@ Access-Control-Allow-Headers | Authorization, Content-Type
 Access-Control-Allow-Methods | GET, POST, OPTIONS
 Access-Control-Allow-Origin | *
 Access-Control-Expose-Headers | Origin, Accept, Content-Location, Location, X-Requested-With
+
+## Extensions
+
+The specification is not prescriptive about support for extensions. However, to support extensions, the specification reserves the name `extension` and will never define an element with that name, allowing implementations to use it to provide custom behavior and information. The value of an extension element should be a pre-coordinated JSON object.
+
+For example, an extension on a request could look like this:
+
+```json
+{
+   "hookInstance" : "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
+   "fhirServer" : "http://hooks.smarthealthit.org:9080",
```

Comment:
For the `fhirServer`, let's use fhir.example.org as the domain (for the same reasons as below).